index-cn.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf8">
+<title>活动守则</title>
+<style>
+* {
+  font-family: "Microsoft YaHei",  "微软雅黑体", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-weight: 200;
+}
+
+html {
+  background: #efefef;
+}
+
+body {
+  background: white;
+  color: #212121;
+  margin: 0;
+  padding-top: 60px;
+}
+
+p, h1, h2 {
+  max-width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+h1 {
+  font-size: 38px;
+}
+
+h2 {
+  font-size: 28px;
+  margin-top: 48px;
+}
+
+p {
+  line-height: 27px;
+  font-size: 18px;
+  text-align: justify;
+}
+
+a {
+  color: blue;
+}
+
+em {
+  font-weight: 400;
+}
+
+.footer {
+  padding: 40px 0;
+  background: #efefef;
+  margin: 0;
+  margin-top: 70px;
+  max-width: 100%;
+}
+
+.footer p {
+  text-align: left;
+}
+</style>
+</head>
+<body>
+  <h1>活动守则</h1>
+
+  <p>所有参与本次会议的出席者，演讲者，赞助人，志愿者均应同意遵守下列行为规范。活动组织者将在活动过程中执行这些规范。我们期望所有参与者共同合作，提供一个安全的环境给大家。</p>
+
+<p><em>简而言之：友善待人</em></p>
+
+<h2>需要帮助？</h2>
+
+<p>在寄给您的 email 中可以找到我们的联络方式。</p>
+
+<h2>简短版</h2>
+
+<p>本会议致力于提供一个无骚扰的会议环境给每一位参与者，无论其性别、年龄、性倾向、残疾、外貌、身材、种族或信仰。我们绝不容忍任何人对会议参与者以任何形式的骚扰。在任何的会议场地中使用含有性的言语及意向在本会议中是不恰当的，包括在讲座中、工作坊、派对上、Twitter 上或任何的网络媒体中。与会者若违反规定，将受到制裁或被驱逐离开会议并<em>不予退款</em>，处置由活动组织者裁定。</p>
+
+<h2>完整版</h2>
```

Comment:
Consider retaining the quick/not so quick contrast. Not as interesting to say "short/full".

Reply:
I don't fully understand what needs to be done here. Can you be more specific please?